ignite/distributed/comp_models/horovod.py

```diff
@@ -2,10 +2,12 @@
 from typing import Any, Callable, Mapping, Optional, Tuple, cast
 
 import torch
+from packaging import version
```

Comment:
packaging is not a built-in dependency and maybe absent in clean envs

Reply:
Thanks for the hint @vfdev-5, so currently the best practice to check version is to use  `LooseVersion` ?

Reply:
Yes, let's use now distutils LooseVersion (you can find the usage in the codebase) and we'll change later